src/cryptoadvance/specterext/spectrum/bridge_rpc.py

```diff
@@ -0,0 +1,83 @@
+import datetime
+import errno
+import json
+import logging
+import os
+import sys
+
+import requests
+import urllib3
+
+from cryptoadvance.specter.helpers import is_ip_private
+from cryptoadvance.specter.specter_error import SpecterError, handle_exception
+from cryptoadvance.specter.rpc import BitcoinRPC
+from cryptoadvance.specter.specter_error import BrokenCoreConnectionException
+
+from cryptoadvance.spectrum.spectrum import RPCError, Spectrum
+
+logger = logging.getLogger(__name__)
+
+# TODO: redefine __dir__ and help
+
+class BridgeRPC(BitcoinRPC):
+    ''' A class which behaves like a BitcoinRPC but internally bridges to Spectrum.jsonrpc '''
+
+    def __init__(
+        self,
+        spectrum,
+        wallet_name=None
+    ):
+        self.spectrum: Spectrum = spectrum
+        self.wallet_name = wallet_name
+
+    def wallet(self, name=""):
+        return type(self)(
+            self.spectrum,
+            wallet_name=name,
+        )
+
+    def clone(self):
+        """
+        Returns a clone of self.
+        Useful if you want to mess with the properties
+        """
+        return self.__class__(
+            self,
+            self.spectrum,
+            wallet=self.wallet
+        )
+
+    def multi(self, calls: list, **kwargs):
+        """Makes batch request to Core"""
+        if self.spectrum is None:
+            raise BrokenCoreConnectionException
+        type(self).counter += len(calls)
+        # some debug info for optimizations
+        # methods = " ".join(list(dict.fromkeys([call[0] for call in calls])))
+        # wallet = self.path.split("/")[-1]
+        # print(f"{self.counter}: +{len(calls)} {wallet} {methods}")
```

Comment:
for debugging, I recommend defining a function like

```
def debug(str)
  if DEBUG_MODE
     print(str)
     ```
     
then you can leave `debug('whatever')` calls all over the codebase and control it with a single global variable
even better is to just use log levels and do debug level logs that you can flag on and off

leaving commented code is not ideal because nobody knows why it's commented and whether it is important

Reply:
We already use a quite reasonable logging-framework but sometimes we're using it not properly. Good hint.